contracts/src/starknet/rewaave/token_bridge.cairo

```diff
@@ -7,31 +7,11 @@ from starkware.cairo.common.uint256 import Uint256
 from starkware.starknet.common.messages import send_message_to_l1
 from starkware.starknet.common.syscalls import get_caller_address, get_contract_address
 
-from rewaave.tokens.IERC20 import IERC20
+from src.starknet.rewaave.tokens.IERC20 import IERC20
```

Comment:
if you use `yarn compile` to compile the cairo contracts, you won't need the `src.starknet` prefix